packages/webcert/src/feature/certificate/CertificateSidePanel/CertificateSidePanel.tsx

```diff
@@ -1,70 +1,78 @@
-  const questionsPanelActive = getResourceLink(resourceLinks, ResourceLinkType.QUESTIONS)
-  const questionsNotAvailablePanelActive = getResourceLink(resourceLinks, ResourceLinkType.QUESTIONS_NOT_AVAILABLE)
+  const resourceLinksForTabs = [
+    ResourceLinkType.SRS,
+    ResourceLinkType.FMB,
+    ResourceLinkType.QUESTIONS,
+    ResourceLinkType.QUESTIONS_NOT_AVAILABLE,
+  ]
 
   if (showSpinner) return null
 
   const handleTabChange = (value: number): void => {
     setSelectedTabIndex(value)
   }
 
-  const getTabs = () => {
-    const tabsArray: ReactNode[] = []
-    const tabsContentArray: ReactNode[] = []
+  const getTab = (name: string, description: string, icon?: ReactNode) => {
+    return (
+      <div data-tip={description}>
+        <p>
+          {icon && icon}
+          {name}
+        </p>
+      </div>
+    )
+  }
 
-    if (questionsPanelActive) {
-      tabsArray.push(
-        <div data-tip={questionsPanelActive.description}>
-          <p>{questionsPanelActive.name}</p>
-        </div>
-      )
+  const getTabFromLink = (link: ResourceLink, icon?: ReactNode) => {
+    return getTab(link.name, link.description, icon)
+  }
 
-      tabsContentArray.push(<QuestionPanel />)
+  const getIcon = (type: ResourceLinkType) => {
+    if (type === ResourceLinkType.FMB || type === ResourceLinkType.SRS) {
+      return <LightbulpIcon className="iu-mr-200" />
     }
+  }
 
-    if (questionsNotAvailablePanelActive) {
-      tabsArray.push(
-        <div data-tip={questionsNotAvailablePanelActive.description}>
-          <p>{questionsNotAvailablePanelActive.name}</p>
-        </div>
-      )
-
-      tabsContentArray.push(<QuestionNotAvailablePanel />)
+  const getPanel = (type: ResourceLinkType) => {
+    switch (type) {
+      case ResourceLinkType.FMB:
+        return <FMBPanel></FMBPanel>
+      case ResourceLinkType.QUESTIONS:
+        return <QuestionPanel />
+      case ResourceLinkType.QUESTIONS_NOT_AVAILABLE:
+        return <QuestionNotAvailablePanel />
+      case ResourceLinkType.SRS:
+        return <SRSPanel />
     }
+  }
 
-    if (fmbInfoPanelActive) {
-      tabsArray.push(
-        <div data-tip={fmbInfoPanelActive.description}>
-          <p>
-            <LightbulpIcon className="iu-mr-200" />
-            {fmbInfoPanelActive.name}
-          </p>
-        </div>
-      )
-
-      tabsContentArray.push(<FMBPanel />)
-    }
+  const getTabs = () => {
+    const tabsArray: ReactNode[] = []
+    const tabsContentArray: ReactNode[] = []
 
-    tabsArray.push(
-      <div data-tip={'Läs om intyget.'}>
-        <p>Om intyget</p>
-      </div>
-    )
+    resourceLinksForTabs.forEach((type) => {
+      const link = resourceLinks.find((l) => l.type === type)
+      if (link) {
+        tabsArray.push(getTabFromLink(link, getIcon(link.type)))
+        tabsContentArray.push(getPanel(link.type))
+      }
+    })
 
+    tabsArray.push(getTab('Om intyget', 'Läs om intyget'))
```

Comment:
Good refactoring here, but I think we can take it a bit further.

We can create a list with resource links for tabs:
```typescript
const resourceLinksForTabs = [
  ResourceLinkType.SRS,
  ResourceLinkType.FMB,
  ResourceLinkType.QUESTIONS,
  ResourceLinkType.QUESTIONS_NOT_AVAILABLE,
]
const availableTabs = resourceLinks.filter(({ type }) => resourceLinksForTabs.includes(type))
```
Then we can remove the `getTabs` function and simply do:

```typescript
return (
  <Tabs
    selectedTabIndex={selectedTabIndex}
    setSelectedTabIndex={handleTabChange}
    tabs={[
      ...availableTabs.map(({ type, name, description }) => getTab(name, description, getIcon(type))),
      getTab('Om intyget', 'Läs om intyget'),
    ]}
    tabsContent={[...availableTabs.map(({ type }) => getPanel(type)), <AboutCertificatePanel key={'about'} />]}
  />
)
```